### DISCUSSION TITLE
The binding IP of monitor

### DISCUSSION THREAD

**Discussion by zhougy1989** [naver/ngrinder#891] [Q&A]
The binding IP of monitor is server IP or controller IP, then public IP or intranet IP
![image](https://user-images.githubusercontent.com/50607317/180930421-aeb11fdb-7a9c-43f8-80c6-4830f7349864.png)
When I bind the public IP of the monitoring server, a link rejection error will appear. If I bind the intranet IP, I will not be able to connect
![image](https://user-images.githubusercontent.com/50607317/180930542-490f99f1-a16d-42ce-bfe6-aee0e3d92ed3.png)
@junoyoon

**Comment by junoyoon** [comment]
@donggyu04 check plz

**Reply by zhougy1989** [reply]
I checked that the problem of null should be the problem of network policy. I want to know whether there are other places to access besides the control machine accessing the 13243 port of the machine where the monitor is located

@donggyu04

**Reply by zhougy1989** [reply]
Do the public IP of the control machine and the public IP of the monitor service need to be in the same domain？
